ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [etc...]
+
+**Expected behavior:** [What you expect to happen]
+
+**Actual behavior:** [What actually happens]
+
+**Reproduces how often:** [What percentage of the time does it reproduce?]
+
+### Versions
+
+Please report the outputs of `psi4 --version`, `python --version`, `numpy --version`, and `scipy --version`.
+
+### Additional Information
+
+Any additional information, configuration or data that might be necessary to reproduce the issue.
```

Comment:
I think this is far too complex. Do you have a concrete reason to add a issue template? They are not used often AFAIK.